### PR TITLE
Adding audible traffic alerts feature

### PR DESCRIPTION
**Pull request by shanelenagh**
This is an initial implementation of an audible ADSB traffic alerts feature.  It includes base announcement of relative clock-position (+high/level/low) of traffic within a configured radius, as well as optional Time to Closest Point of Approach (TCPA) feature, also configurable with a "closest" distance and time threshold before announcement is spoken.  There is a recently added "max alert frequency" feature to prevent the alerts from being too chatty/repetitive with announcing the same traffic (by ADSB callsign) repeatedly.  I intend to do a nice Venn diagram for the base audible alerts announcements vs. the more specific TCPA closing alerts vs. the even more specific sawtooth double-chirp when the TCPA is within a configured percentage.

From a technical perspective, the individual audio files (mono MP3's, with my voice for now, haha) are played by a sequential audio player using the Android SoundPool mechanism for fast in-memory audio playbook, most commonly used in games.  I also run the alert processor in a separate thread, to keep the UI from lagging due to this feature.  Memory management and GC tests have been conducted to ensure reasonable feature memory usage and appropriate GC of objects, but there is always more that could be optimized there, no doubt.

### CODE REVIEW

File: app/src/main/java/com/ds/avare/views/LocationView.java [hunk 1]
```diff
@@ -944,6 +965,7 @@ private void drawMap(Canvas canvas) {
         drawDrawing(canvas, ctx);
         drawCapGrids(canvas, ctx);
         drawTraffic(canvas, ctx);
+        handleAudibleAlerts();
```
Comment (apps4av) [inline app/src/main/java/com/ds/avare/views/LocationView.java:968]: This is drawing context. Suggest moving this to a separate non ui thread

**Review by apps4av — COMMENTED** [review]
The changes seem quite straightforward. Whats the copyright for the mp3 files included ?

**Comment by apps4av** [comment]
Tested and seems to be working well. Some issues:
Keep getting ownship notifications.
Cannot disable audio notifications. Even though, disabled in Preferences, still getting audible alerts.

**Review by apps4av — APPROVED** [review]

**Comment by shanelenagh** [comment]
> The changes seem quite straightforward. Whats the copyright for the mp3 files included ? 

Yeah, I really don't care about those. If you want something written saying I assigned copyright to you that's fine. Nothing special in those recordings, haha. I actually thought about doing them all the way up to 60 seconds, and I have the Audacity recording that has that, but SoundPool loads all of those into memory uncompressed, so I didn't want to bloat the memory size of the feature too much.

**Comment by apps4av** [comment]
Was asking if the files are entirely yours or if you copied them from
somewhere. If you made them then it's fine. The PR is merged to master with
some adjustments.

**Comment by shanelenagh** [comment]
The voice files are all my voice, yes, and I recorded them with Audacity
and a Blue Snowball mic--nothing fancy. The sawtooth double chirp I also
created with Audacity, using their chirp plug-in with parameters I set.

**Pull request merged**